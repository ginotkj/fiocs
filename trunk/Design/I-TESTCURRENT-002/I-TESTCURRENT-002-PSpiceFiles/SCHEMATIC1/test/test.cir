** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-002\i-testcurrent-002-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_10.5\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 100p 0.1p 0.01p 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
