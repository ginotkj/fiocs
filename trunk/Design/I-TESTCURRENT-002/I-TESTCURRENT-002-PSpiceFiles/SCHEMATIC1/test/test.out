
**** 11/11/09 12:27:21 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-002\i-testcurrent-002-pspicefiles\schematic1\test.sim


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_10.5\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 100p 0.1p 0.01p 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source I-TESTCURRENT-002
I_I1         N14389 0 DC 0Adc AC 0Aac
+EXP 0m 1m 1.5p .5p 4p 1p
R_R1         N14389 N14398  1k TC=0,0 
R_R2         0 N14398  1k TC=0,0 

**** RESUMING test.cir ****
.END

**** 11/11/09 12:27:21 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-002\i-testcurrent-002-pspicefiles\schematic1\test.sim


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N14389)    0.0000 (N14398)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT


    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 11/11/09 12:27:21 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-002\i-testcurrent-002-pspicefiles\schematic1\test.sim


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .86
